cnf-tests/testsuites/e2esuite/security/tuning.go

```diff
@@ -152,16 +160,19 @@ var _ = Describe("[tuningcni]", func() {
 		})
 
 		AfterEach(func() {
-			updateAllowlistConfig(originalSysctls)
+			err := client.Client.Delete(context.Background(), macVlandNad)
```

Comment:
Does it panic if `macVlandNad` is nil? What about using `DeferCleanup(...)` after line 176?

Reply:
I'm not sure if it panics with nil. Nevertheless, thanks for the suggestion. I'll use `DeferCleanup`.